member-service/src/main/java/com/hedvig/memberservice/aggregates/BisnodeAddress.java

```diff
@@ -17,4 +24,24 @@
   private String city;
   private String country;
   private List<String> formattedAddress;
+
+    public static int parseFloorFromApartment(String apartmentNo) {
+
+      if(apartmentNo == null){
+        return 0;
+      }
+
+      final Pattern compile = Pattern.compile("\\d\\d\\d\\d");
+      final Matcher matcher = compile.matcher(apartmentNo == null ? "" : apartmentNo);
```

Comment:
```suggestion
      final Matcher matcher = compile.matcher(apartmentNo);
```

We have already null-checked `apartmentNo` at this point 🙂